### ISSUE TITLE
Unable to add NIC to VM - Insufficient capacity when adding NIC to VM

### ISSUE THREAD

**Issue by mdesaive** [apache/cloudstack#2592]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.11.0.0
~~~

##### OS / ENVIRONMENT

Advanced Networking, Cloudstack 4.11.0.0 on Ubuntu 14.04 LTS,
MySQL 5.5.59-0ubuntu0.14.04.1

##### SUMMARY

On one of our two CloudStack instances we have an issue adding NICs to
VMs through the Web-UI after upgrading to 4.11.0.0. On another instance the problem does
not occur after the upgrade. I already asked on users@cloudstack to reproduce the issue, but got no response. So this might be an individual problem.

Adding a NIC with CloudMonkey through a command like the following ist working perfectly:

create network displaytext=deleteme-cloudmonkey
name=deleteme-cloudmonkey networkofferingid=<id> zoneid=<id>
projectid=<id> gateway=172.17.1.1 netmask=255.255.252.0
networkdomain=deleteme-cloudmonkey.heinlein-intern.de

This issue maybe is close to #2533 "Web-UI creates all isolated Nets with IP range 10.1.1.0/24" - in both cases the API call through CloudMonkey works fine, but using the web-UI leads to an error. Maybe it could make sense to investigate those two issues together.

##### STEPS TO REPRODUCE

~~~
Create a new isolated network
Wait until network is "implemented"
Add NIC on this new network to a VM
~~~

##### EXPECTED RESULTS

~~~
To have the NIC added to VM and network. ;)
~~~

##### ACTUAL RESULTS

~~~
=> Textbox: Insufficient capacity when adding NIC to VM[User|i-<id>-VM]:
com.cloud.exception.InsufficientAddressCapacityException: Insufficient
address capacityScope=interface com.cloud.dc.DataCenter; id=1
=> No new NIC is added to the VM
~~~

**Comment by PaulAngus** [comment]
Hi @mdesaive which version of CloudStack were you upgrading from?

**Comment by rhtyd** [comment]
I could not reproduce this issue on latest 4.11 branch. I did this:
- I added a new isolated network, it went into "Allocated" state
- On an already deployed/running VM, I add a nic on the newly created network to it
- This kicked a new VR for that new network and a nic was successfully added, no errors see.

![screenshot from 2018-05-09 16-20-11](https://user-images.githubusercontent.com/95203/39811021-03e4e2ca-53a5-11e8-8780-d70b07e5f7b3.png)
![screenshot from 2018-05-09 16-20-21](https://user-images.githubusercontent.com/95203/39811024-04272d10-53a5-11e8-9662-0b8230368fd4.png)

Closing this issue as I could not reproduce it.
